docs/data/material/components/tabs/tabs.md

```diff
@@ -71,6 +71,12 @@ By default, left and right scroll buttons are automatically presented on desktop
 
 {{"demo": "ScrollableTabsButtonAuto.js", "bg": true}}
 
+### Dynamically added tabs
+
+You can add and remove tabs dynamically and the scroll buttons will automatically adjust based on the available space.
```

Comment:
Do we really need the demo? If it is just for testing purposes, we could just create codesandbox. Does it bring something to the user? I can't see anything specific in this that is bringing some value.

Reply:
I added it to just show the fix. The plan was to remove before merge.